tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -367,15 +374,19 @@ class Esdl2EclCmd : public EsdlConvertCmd
                 "esdl ecl sourcePath outputPath [options]\n"
                 "\nsourcePath must be absolute path to the ESDL Definition file containing the"
                 "EsdlService definition for the service you want to work with.\n"
-                "outputPath must be the absolute path where the ECL output with be created."
+                "outputPath must be the absolute path where the ECL output with be created.\n"
                 "   Options:\n"
                 "      -x, --expandedxml     Output expanded XML files\n"
                 "      --includes            Process all included files\n"
-                "      --rollup              Roll-up all processed includes to single ecl output file"
+                "      --rollup              Roll-up all processed includes to single ecl output file\n"
+                "      --CDE, --compfilesdir HPCC Component files directory (xslt files)\n"
+                "      --ecl-includes        Coma-delimited import list to be attached to output ECL\n"
```

Comment:
Command line support is good, but still think a new ESDL keyword would be more user freindly so they don't have to type it each time.  ECLImport("xxxx"). for example.  Or use or modify the keyword that already exists... I know there is something similar used for the generated c++.